src/Umbraco.Cms.Integrations.DAM.Aprimo/Models/ViewModels/AprimoAssetViewModel.cs

```diff
@@ -9,6 +9,14 @@ public class AprimoAssetViewModel
 
         public string Thumbnail { get; set; } = string.Empty;
 
-        public string Tag { get; set; } = string.Empty;
+        public IEnumerable<AprimoCropItemViewModel> CropItemsVM { get; set; }
```

Comment:
Not sure we need the `VM` suffix here, as it doesn't seem to be consistent with other view model fields.  Why not just `CropItems` (or `Crops`) and `Fields`?